lib/Basics/PhysicalMemory.cpp

```diff
@@ -100,7 +100,7 @@ struct PhysicalMemoryCache {
       if (!value.empty()) {
         uint64_t multiplier = 1;
         if (value.back() == 'G' || value.back() == 'g') {
-          multiplier = 1024*1024*1024;
+          multiplier = 1024 * 1024 * 1024;
           value.pop_back();
         } else if (value.back() == 'M' || value.back() == 'm') {
```

Comment:
This may confuse users. ArangoDB has options to specify memroy in `G` and `GiB`.
Where G is power of 10 and `GiB` is power of 2.
But here `G` is power of 2.